server/lib/get-browsers.js

```diff
@@ -11,15 +11,16 @@ let bv = importJSON('../node_modules/browserslist/package.json').version
 let cv = importJSON('../node_modules/caniuse-lite/package.json').version
 let updated = statSync(join(ROOT, '../../../pnpm-lock.yaml')).mtime
 
-export async function getBrowsers(query, region) {
+export async function getBrowsers(rawQuery, region) {
   let browsersByQuery = []
+  let preparedQuery = prepareQuery(rawQuery)
```

Comment:
Acceptable!

Reply:
Btw, it also seems correct to rename entities on the client side, too. 
What we treat as a `query` should be a `config`. Will help to avoid some confusion.

Reply:
I like it. We can even change the API (but need to change docs, of course).